src/App.jsx

```diff
@@ -14,45 +17,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_TYPE_LENG = 'leng';
+const SORT_TYPE_ALPH = 'alph';
+const REVERSE_BASE_STATE = false;
+
+const updateGoods = (goods, { sortType, isReverse }) => {
```

Comment:
Not critical - still a small extra point

As long as you are copying the array - I would check the input parameters to return the input without any copying, if no manipulations needed

`
if (sortType === '' & !reverse) {
  return goods
}
`

You can even add a constant
```
const SORT_TYPE_NONE = ''
```